docs/guides/bake/file-definition.md

```diff
@@ -0,0 +1,363 @@
+---
+title: "Bake file definition"
+keywords: build, buildx, bake, buildkit, hcl, json, compose
+---
+
+`buildx bake` supports HCL, JSON and Compose file format for defining build
+groups and targets. It looks for build definition files in the current
+directory in the following order:
+
+* `docker-compose.yml`
+* `docker-compose.yaml`
+* `docker-bake.json`
+* `docker-bake.override.json`
+* `docker-bake.hcl`
+* `docker-bake.override.hcl`
+
+A target reflects a single docker build invocation with the same options that
```

Comment:
This should start with an intro like "Inside a bake file you can define group, target and variable blocks". 

Also mention that "when you invoke `bake` you specify what targets/groups you want to build. If no arguments is specified group/target named "default" will be built"

Reply:
^